app/views/home.scala.html

```diff
@@ -65,39 +65,47 @@ <h1>
             <span id="sub-heading" class="heading-secondary"><span id="ref">@messages("home.ref")</span><span id="it-reference"> @user.mtditid</span></span>
         </h1>
 
-        <div class="grid-row">
-
-            <div id="bills-section" class="column-one-half">
-                <h3 id="bills-heading" class="heading-small">@messages("home.bills.heading")</h3>
-                <p id="bills-text">@messages("home.bills.text")</p>
-                <p><a id="bills-link" href="@{controllers.routes.BillsController.viewCrystallisedCalculations().url}">@messages("home.bills.link")</a></p>
-            </div>
+        <div class="flex-container grid-row">
```

Comment:
This new class is causing the sections on the home page to stack differently to the prototype.